test/functional/services/dashboard/panel_actions.ts

```diff
@@ -43,9 +47,14 @@ export class DashboardPanelActionsService extends FtrService {
 
   async toggleContextMenu(parent?: WebElementWrapper) {
     this.log.debug(`toggleContextMenu(${parent})`);
-    await (parent ? parent.moveMouseTo() : this.testSubjects.moveMouseTo('dashboardPanelTitle'));
+    if (parent) {
+      await parent.scrollIntoViewIfNecessary(DASHBOARD_TOP_OFFSET);
+      await this.browser.getActions().move({ x: 0, y: 0, origin: parent._webElement }).perform();
+    } else {
+      await this.testSubjects.moveMouseTo('dashboardPanelTitle');
+    }
```

Comment:
The `web_element_wrapper` service does not accept a `topOffset` for their `moveMouseTo` function - this was causing issues because, at the top of this function, they call `scrollIntoViewIfNecessary` **without** taking in to account the necessary offset. This was causing the top of the panels to be overlapped by the sticky top nav bar, even if they were already in view.

Rather than requesting a review from the QA team to make their version of `moveMouseTo` accept a top offset for the scroll, I decided to just recreate the actions here instead :shrug: